typescript/infra/config/environments/mainnet3/warp/configGetters/getBscMilkywayMILKWarpConfig.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  ChainMap,
+  HypTokenRouterConfig,
+  IsmType,
+  MultisigConfig,
+  TokenType,
+  buildAggregationIsmConfigs,
+} from '@hyperlane-xyz/sdk';
+import { Address } from '@hyperlane-xyz/utils';
+
+import { RouterConfigWithoutOwner } from '../../../../../src/config/warp.js';
+
+// TODO: Confirm ownership
+const safeOwners: ChainMap<Address> = {
+  bsc: '0xa7ECcdb9Be08178f896c26b7BbD8C3D4E844d9Ba',
+  milkyway: '0xa7ECcdb9Be08178f896c26b7BbD8C3D4E844d9Ba',
+  ism: '0xa7ECcdb9Be08178f896c26b7BbD8C3D4E844d9Ba',
```

Comment:
can delete the `ism` line - since the bsc ism owner should just be the bsc owner